kobuki_maze:
  # The maze task
  action_space: 2
  observation_space: 15 # 8 + 2 + 3 + 2 
  lidar_samples: 8

  #-----------------------------------------------------------------------------------------#
  #--- Simulation parameters
  
  time_step: 0.01
  update_rate_multiplier: 10.0 # If training multiply the max update rate by this number 

  #-----------------------------------------------------------------------------------------#
  #--- Set goal pos parameters

  tolerance_goal_pos: 0.50 # Tolerance for the position of the goal

  #-----------------------------------------------------------------------------------------#
  #--- Set variables related to rewards
  
  step_reward: 0.0
  multiplier_dist_reward: 10.0
  angle_diff_reward_multiplier: 4.0
  reached_midway_reward:  10.0
  not_reached_midway_reward: -1.0
  reached_goal_reward: 100.0
  collision_reward: -200.0 # If collision with the wall
  collision_distance_threshold: 0.25 # If collision with the wall